search/src/pages/create.astro

```diff
@@ -4,9 +4,6 @@ import CreateNewEvidenceForm from "../components/CreateNewEvidenceForm";
 import Footer from "../components/Footer.astro";
 ---
 
-<head>
-  <script src="/tinymce/tinymce.min.js" :inline></script>
-</head>
```

Comment:
We cannot remove this. It is needed for the create card and edit card forms.

Reply:
ooh got it